sqlglot/optimizer/qualify_columns.py

```diff
@@ -141,9 +141,9 @@ def validate_qualify_columns(expression: E) -> E:
 
 def _unpivot_columns(unpivot: exp.Pivot) -> t.Iterator[exp.Column]:
     name_column = []
```

Comment:
You could do this in a single pass. Should we rename to use plural now that there are multiple fields?
```suggestion
    name_column = [
        field.this
        for field in unpivot.fields
        if isinstance(field, exp.In) and isinstance(field.this, exp.Column)
    ]
```